src/olympia/api/paginator.py

```diff
@@ -21,6 +21,8 @@ def validate_number(self, number):
             raise PageNotAnInteger('That page number is not an integer')
         if number < 1:
             raise EmptyPage('That page number is less than 1')
+        elif number > 100000:
```

Comment:
I tried `sys.maxint` and it didn't work. First, it's actually much larger than `2^^32 - 1` (`9223372036854775807` on my machine), and second, even `2^^32 - 1` didn't work either, it failed before that. Which is why I settled on a lower, more closer than the realistic limit we'll ever need, value.